lib/attrsets.nix

```diff
@@ -534,23 +534,42 @@ rec {
         in mapAttrs g;
     in recurse [] set;
 
-
   /* Generate an attribute set by mapping a function over a list of
      attribute names.
 
      Example:
-       genAttrs [ "foo" "bar" ] (name: "x_" + name)
-       => { foo = "x_foo"; bar = "x_bar"; }
+       mapListToAttrs [ "foo" "bar" ] (name: "x_" + name)
+       => { bar = "x_bar"; foo = "x_foo"; }
 
      Type:
-       genAttrs :: [ String ] -> (String -> Any) -> AttrSet
+       mapListToAttrs :: [String] -> (String -> Any) -> AttrSet
```

Comment:
None of this documentation takes into account the flipped order.

Reply:
Fixed in 195c0a4